package.json

```diff
@@ -68,6 +76,8 @@
   "devDependencies": {
     "@contentful/browserslist-config": "^2.0.0",
     "@contentful/rich-text-types": "^15.3.6",
+    "@parcel/packager-ts": "^2.0.0-nightly.982",
```

Comment:
It would be great to have a comment in the PR description why we pinning a nightly build here ;)

Reply:
As the nightly build didn't solve our problem, I think we should use `@latest` instead.